app/views/about/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>About Us:</h1>
+
+<% index = 1 %>
+<% @about_info.each do |about_me| %>
+		<div class="about-<%= index %>">
+			<h3><%= about_me.name %></h3>
+			<%= image_tag("#{about_me.image}", width: 200) %>
```

Comment:
This appears to not be working when adding a `save_and_open_page` line to the tests, but it is working when running `rails s` and going to `localhost:3000`!